lib/0uint.js

```diff
@@ -3,7 +3,14 @@
 import { Token, Type } from './token.js'
 import { decodeErrPrefix, assertEnoughData } from './common.js'
 
-export const uintBoundaries = [24, 2 ** 8, 2 ** 16, 2 ** 32, BigInt(2) ** BigInt(64)]
+export const uintBoundaries = [
+  24,
+  2 ** 8,
+  2 ** 16,
+  2 ** 32,
+  // BigInt(2) ** BigInt(64) gets transpiled to Math.pow(BigInt(2), BigInt(64)) so use the literal value
+  18446744073709551616n
```

Comment:
I can't use literals though, I started off with all literals but it breaks with some version of something, I can't recall which, but literals are a bit too new compared to the constructor, `BigInt('18446744073709551616')` might be acceptable though.